src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -29,6 +35,8 @@ interface DropdownMenuGeneralProps<T> {
     defaultSwitcherClassName?: string;
     /** Invoked whenever the switcher was clicked (if DropdownMenu is not disabled) */
     onSwitcherClick?: (event?: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+    /** Allows to override some props for a default dropdown Menu */
+    defaultMenuProps?: MenuProps;
```

Comment:
There is no menu other than this, `default` can be omitted.

Reply:
Fixed